services/QuillLMS/spec/models/districts_admins_spec.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: districts_admins
+#
+#  id          :bigint           not null, primary key
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#  district_id :bigint           not null
+#  user_id     :bigint           not null
+#
+# Indexes
+#
+#  index_districts_admins_on_district_id  (district_id)
+#  index_districts_admins_on_user_id      (user_id)
+#
+require 'rails_helper'
+
+describe DistrictsAdmins, type: :model, redis: true do
+  it { should belong_to(:district) }
+  it { should belong_to(:user) }
+
+  it { is_expected.to callback(:attach_to_subscribed_schools).after(:create) }
+  it { is_expected.to callback(:detach_from_schools).after(:destroy) }
+
+  let(:user) { create(:user, email: 'test@quill.org') }
+  let(:district) { create(:district) }
+  let(:school) { create(:school, district: district) }
+  let(:subscription) { create(:subscription, account_type: 'School Paid') }
+  let(:admins) { create(:districts_admins, user: user, district: district) }
+
+  describe '#admin' do
+    it 'should return the user associated' do
+      expect(admins.admin).to eq(admins.user)
+    end
+  end
+
+  describe '#attach_to_subscribed_schools' do
```

Comment:
You may want a case here for situations where someone is already an admin for a school when this gets triggered?  Like if they were manually assigned or something like that?